app/javascript/components/editor/header/ActionMore.tsx

```diff
@@ -1,7 +1,7 @@
-import React, { useCallback, useEffect, useState } from 'react'
+import React, { useCallback, useState } from 'react'
```

Comment:
Enjoying the "remove unused usings" refactoring of VS Code @kntsoriano? :)

Reply:
Yes!!!!